Build fail

When building the project, running `cmake --build build -j 1`, it gives me the following error message:
`
home/pedrodemargomes/pedro/donut/src/Render/SkinAnimation.h: In member function ‘T Donut::SkinAnimation::ValueKeyCurve<T>::Evalulate(float, T)’:
/home/pedrodemargomes/pedro/donut/src/Render/SkinAnimation.h:79:11: error: ‘glm’ has not been declared
   79 |    time = glm::mod(time, _keyValues[lastIndex]->GetTime());
      |           ^~~
make[2]: *** [src/CMakeFiles/donut.dir/build.make:63: src/CMakeFiles/donut.dir/AnimCamera.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:94: src/CMakeFiles/donut.dir/all] Error 2
make: *** [Makefile:84: all] Error 2
`

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -50,6 +50,13 @@ target_link_libraries(${PROJECT_NAME}
 		fmt::fmt
 	)
 
+if (UNIX)
+	target_link_libraries(${PROJECT_NAME}
+			PRIVATE
+			${CMAKE_DL_LIBS}
+		)
+endif()
+
 # configure filesystem for slightly older compilers
 if (_CXX_FILESYSTEM_HAVE_HEADER)
 	target_compile_definitions(${PROJECT_NAME} PRIVATE DONUT_HAS_FILESYSTEM)

--- a/src/Core/MemoryStream.h
+++ b/src/Core/MemoryStream.h
@@ -4,6 +4,7 @@
 
 #include <string>
 #include <vector>
+#include <cstring>
 
 namespace Donut
 {

--- a/src/P3D/P3DChunk.h
+++ b/src/P3D/P3DChunk.h
@@ -8,6 +8,7 @@
 #include <fmt/format.h>
 #include <memory>
 #include <vector>
+#include <iostream>
 
 namespace Donut::P3D
 {

--- a/src/Render/SkinAnimation.h
+++ b/src/Render/SkinAnimation.h
@@ -9,6 +9,7 @@
 #include <memory>
 #include <string>
 #include <vector>
+#include <glm/glm.hpp>
 
 namespace Donut
 {

